posthog/api/event.py

```diff
@@ -62,14 +62,8 @@ class Meta:
 
     def get_person(self, event: Event) -> Any:
         if hasattr(event, "person_properties"):
-            if event.person_properties:  # type: ignore
-                return event.person_properties.get("email", event.distinct_id)  # type: ignore
-            else:
-                return event.distinct_id
-        try:
-            return event.person.properties.get("email", event.distinct_id)
-        except:
-            return event.distinct_id
+            return event.person_properties  # type: ignore
```

Comment:
I'd call this something else. `person_properties` kind of implies that you're getting `person.properties`, not an object `{ is_identified, distinct_id, properties: {} }`